front/app/modules/commercial/insights/admin/containers/Reports/index.tsx

```diff
@@ -35,11 +35,14 @@ const ReportTab = memo(() => {
     ? projects.data.filter((project) => {
         const processType = project?.attributes.process_type;
         const participationMethod = project.attributes.participation_method;
+
         return (
           (processType === 'continuous' &&
-            !['information', 'survey', 'volunteering', null].includes(
-              participationMethod
-            )) ||
+            participationMethod !== 'information' &&
```

Comment:
Minor and just for my knowledge. Why not use includes? I think it is more concise, easier to read and maintain 🤔

Reply:
Hmm, I found the existing code harder to read when I wrote it, but no strong reason. Put it back to the old version.